### DISCUSSION TITLE
Dynamite Core Team Meeting 5th April

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#374] [Announcements]
Hello @dynamics-of-stellar-systems/dynamite-maintainers,

the next team meeting will be **5th April 1pm**.

A preliminary agenda is:
@prashjet would you be ok to give a brief summary of the geckos meeting with regard to dynamite?
@glennvdven will provide some papers for discrete Schwarzschild modelling to discuss

Please respond, if you will join and/or want to discuss something else as well.

Cheers,
Alina

**Comment by glennvdven** [comment]
Dear all,

For our discussion next week Friday, please find below a few papers. 
The first paper 1a is the most relevant, but had issues for which recent paper 1b might provide the solution. 

Cheers,
Glenn

>>>

1: Axisymmetric Schwarzschild orbit-based fitting DISCRETE, but not recovering inclination... 
a: Chaname et al. (2008): Constraining the Mass Profiles of Stellar Systems: Schwarzschild Modeling of Discrete Velocity Data Sets
https://ui.adsabs.harvard.edu/abs/2008ApJ...682..841C/abstract
b: ...solution in effective degrees of freedom?
- Lipka & Thomas (2021): A novel approach to optimize the regularization and evaluation of dynamical models using a model selection framework
https://ui.adsabs.harvard.edu/abs/2021MNRAS.504.4599L/abstract

2: Axisymmetric Jeans velocity-moment-based fitting DISCRETE:
- Watkins et al. (2013): Discrete dynamical models of ω Centauri
https://ui.adsabs.harvard.edu/abs/2013MNRAS.436.2598W/abstract

3: Spherical distribution-function fitting DISCRETE:
a: Wu & Tremaine (2006): Deriving the Mass Distribution of M87 from Globular Clusters
https://ui.adsabs.harvard.edu/abs/2006ApJ...643..210W/abstract
b: Magorrian (2019): Orbit-superposition models of discrete, incomplete stellar kinematics: application to the Galactic centre
https://ui.adsabs.harvard.edu/abs/2019MNRAS.484.1166M/abstract

<<<

> On 22. Mar 2024, at 13:32, Alina Boecker ***@***.***> wrote:
> 
> 
> Hello @dynamics-of-stellar-systems/dynamite-maintainers <https://github.com/orgs/dynamics-of-stellar-systems/teams/dynamite-maintainers>,
> 
> the next team meeting will be 5th April 1pm.
> 
> A preliminary agenda is:
> @prashjet <https://github.com/prashjet> would you be ok to give a brief summary of the geckos meeting with regard to dynamite?
> @glennvdven <https://github.com/glennvdven> will provide some papers for discrete Schwarzschild modelling to discuss
> 
> Please respond, if you will join and/or want to discuss something else as well.
> 
> Cheers,
> Alina
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/dynamics-of-stellar-systems/dynamite/discussions/374>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJPLFDYKTBAUQAIT4LG53LDYZQQFTAVCNFSM6AAAAABFDHW4IOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQYDQMZUGI>.
> You are receiving this because you were mentioned.
> 

___________________________
Prof. Dr. Glenn van de Ven
Head Department of Astrophysics (astro.univie.ac.at <https://astro.univie.ac.at/en/>)
Director Vienna International School of Earth and Space Sciences (visess.univie.ac.at <https://visess.univie.ac.at/>)
University of Vienna, Türkenschanzstraße 17, 1180 Vienna, Austria
+43 1 4277 53806, https://dynamics.univie.ac.at <https://dynamics.univie.ac.at/>
